### DISCUSSION TITLE
[Question]  How can I use Status Indicators in React ?

### DISCUSSION THREAD

**Discussion by bepyan** [carbon-design-system/carbon#10088] [Help] [answered]
I want to use status indicator in table like this picture.
How can I use it in React?
Do I have to just define the svg file?

![image](https://user-images.githubusercontent.com/65283190/141253230-af6a1840-4b28-442a-9cc1-5c9c9d3f3b0e.png)

**Comment by tay1orjones** [comment, marked as answer]
Hi @bepyan - the status indicator icons are included in the [`@carbon/icons-react`](https://www.npmjs.com/package/@carbon/icons-react) package and can be imported from there for use in your application.

For reference, the Carbon website has an [icon library page](https://www.carbondesignsystem.com/guidelines/icons/library/) where you can view all the icons and their names. Filter it by `Status` to see the different status indicator icons available.

![image](https://user-images.githubusercontent.com/3360588/141332673-1d091cf6-cbf6-4726-9abf-956c0f5f9703.png)
